test/chisel.py

```diff
@@ -0,0 +1,178 @@
+"""
+A simple client that uses the Python ACME library to run a test issuance against
+a local Boulder server. Usage:
+
+$ virtualenv venv
+$ . venv/bin/activate
+$ pip install -r requirements.txt
+$ python chisel.py foo.com bar.com
+"""
+import json
+import logging
+import os
+import sys
+import threading
+import time
+import urllib2
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+
+import OpenSSL
+
+from acme import challenges
+from acme import client as acme_client
+from acme import errors as acme_errors
+from acme import jose
+from acme import messages
+from acme import standalone
+
+logger = logging.getLogger()
+logger.setLevel(int(os.getenv('LOGLEVEL', 20)))
+
+def make_client(email=None):
+    """Build an acme.Client and register a new account with a random key."""
+    key = jose.JWKRSA(key=rsa.generate_private_key(65537, 2048, default_backend()))
+
+    net = acme_client.ClientNetwork(key, verify_ssl=False,
+                                    user_agent="Boulder integration tester")
+
+    client = acme_client.Client("http://localhost:4000/directory", key=key, net=net)
```

Comment:
It might be useful to allow specifying the directory URL via a parameter or overriding it via an env var.